kubernetes/spack/gitlab-spack-io/web/deployment.yaml

```diff
@@ -100,7 +100,11 @@ spec:
             # gitlab_rails['initial_shared_runners_registration_token'] = (
             #     runner_token unless runner_token.to_s == '');
 
-            prometheus_monitoring['enable'] = false
+            prometheus_monitoring['enable'] = true;
+            prometheus['listen_address'] = 'localhost:9090';
+            grafana['disable_login_form'] = false;
+            grafana['admin_password'] = ENV['GITLAB_ROOT_PASSWORD'];
```

Comment:
Is there a web UI for grafana?  How does one access it?

Reply:
We still have some work to do to get all the metric we want, but this is a step in the right direction.

To access grafana:
https://gitlab.spack.io/-/grafana

A dashboard that actually shows something useful:
https://gitlab.spack.io/-/grafana/d/x6Z50y-iz/gitlab-omnibus-gitaly?orgId=1&refresh=1m

One that still needs work:
https://gitlab.spack.io/-/grafana/d/wsSteMemz/gitlab-omnibus-overview